tests/functional/pv/space_reclaim/test_rook_reclaim_namespace.py

```diff
@@ -0,0 +1,127 @@
+        4.Verify reclaimspacejob successful completion
+        5.Check logs of csi-rbdplugin-provisioner-xxx/csi-rbdplugin pods.
+        6.Verify sparsify is skipped.
+        7.Delete pod
+        8.Delete reclaimspacejob CR
+        9.Recreate reclaimspacejob CR
+        10.Sleep 70 seconds so the logs in csi-rbdplugin-provisioner-xxx/csi-rbdplugin will be updated
+        11.Verify logs does not show 'skipping sparsify operation' message.
+        """
+        log.info("Create RBD PVC with filesystem mode")
+        pvc_obj = pvc_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            status=constants.STATUS_BOUND,
+            size=2,
+        )
+        pod_dict_path = constants.NGINX_POD_YAML
+        raw_block_pv = False
+
+        log.info(
+            f"Created new pod sc_name={constants.CEPHFILESYSTEM} size=10Gi, "
+            f"access_mode={constants.ACCESS_MODE_RWX}, volume_mode={constants.VOLUME_MODE_FILESYSTEM}"
+        )
+        pod_obj = pod_factory(
+            interface=constants.CEPHFILESYSTEM,
+            pvc=pvc_obj,
+            status=constants.STATUS_RUNNING,
+            pod_dict_path=pod_dict_path,
+            raw_block_pv=raw_block_pv,
+        )
+
+        log.info(f"Create reclaimspacejob CR to run on pvc {pvc_obj.name}")
+        self.reclaim_job_before_pod_delete = create_reclaim_space_job(
+            pvc_name=pvc_obj.name
+        )
+        expected_log = "skipping sparsify operation"
+        pod_names = get_csi_provisioner_pod(interface=constants.CEPHBLOCKPOOL)
+
+        log.info(
+            f"Check logs of csi-rbdplugin-provisioner-xxx/csi-rbdplugin pods {pod_names}"
+        )
+        sample = TimeoutSampler(
+            timeout=100,
+            sleep=5,
+            func=verify_log_exist_in_pods_logs,
+            pod_names=pod_names,
+            expected_log=expected_log,
+        )
+        if not sample.wait_for_func_status(result=True):
+            raise ValueError(
+                f"The expected log '{expected_log}' does not exist in {pod_names} pods"
+            )
+
+        log.info(f"Delete pod {pod_obj.name}")
+        pod_obj.delete()
+
+        log.info(f"Delete reclaimspacejob CR {self.reclaim_job_before_pod_delete.name}")
+        self.reclaim_job_before_pod_delete.delete()
+
+        log.info("Recreate reclaimspacejob CR")
+        self.reclaim_job_after_pod_delete = create_reclaim_space_job(
+            pvc_name=pvc_obj.name
+        )
+
+        log.info(f"Sleep 70 seconds so the logs in {pod_names} will be updated")
+        time.sleep(70)
+
+        log.info("Verify logs does not show 'skipping sparsify operation' message.")
+        log_exist = verify_log_exist_in_pods_logs(
+            pod_names=pod_names, expected_log=expected_log, since="70s"
+        )
+        if log_exist:
+            raise ValueError(
+                f"The expected log '{expected_log}' exist in {pod_names} pods after reclaimspacejob deletion"
+            )
```

Comment:
can we call the function using Timeoutsampler? that way we done need to hardcode the sleep time

Reply:
it is more difficult because  I need to change the "since" flag each iteration.. In my process , we wait 70 seconds and then verify log does not exist